ui/filewriter_ctrl_frame.py

```diff
@@ -0,0 +1,199 @@
+        self.line = QtWidgets.QFrame(self.centralwidget)
+        self.line.setFrameShape(QtWidgets.QFrame.VLine)
+        self.line.setFrameShadow(QtWidgets.QFrame.Sunken)
+        self.line.setObjectName("line")
+        self.horizontalLayout.addWidget(self.line)
+        self.commandLayout = QtWidgets.QVBoxLayout()
+        self.commandLayout.setContentsMargins(-1, 0, -1, 0)
+        self.commandLayout.setObjectName("commandLayout")
+        self.commandBrokerLayout = QtWidgets.QHBoxLayout()
+        self.commandBrokerLayout.setObjectName("commandBrokerLayout")
+        self.commandBrokerLabel = QtWidgets.QLabel(self.centralwidget)
+        self.commandBrokerLabel.setObjectName("commandBrokerLabel")
+        self.commandBrokerLayout.addWidget(self.commandBrokerLabel)
+        self.commandBrokerEdit = QtWidgets.QLineEdit(self.centralwidget)
+        self.commandBrokerEdit.setObjectName("commandBrokerEdit")
+        self.commandBrokerLayout.addWidget(self.commandBrokerEdit)
+        self.commandLayout.addLayout(self.commandBrokerLayout)
+        self.line_3 = QtWidgets.QFrame(self.centralwidget)
+        self.line_3.setFrameShape(QtWidgets.QFrame.HLine)
+        self.line_3.setFrameShadow(QtWidgets.QFrame.Sunken)
+        self.line_3.setObjectName("line_3")
+        self.commandLayout.addWidget(self.line_3)
+        self.commandFormLayout = QtWidgets.QFormLayout()
+        self.commandFormLayout.setFieldGrowthPolicy(QtWidgets.QFormLayout.ExpandingFieldsGrow)
+        self.commandFormLayout.setLabelAlignment(QtCore.Qt.AlignRight|QtCore.Qt.AlignTrailing|QtCore.Qt.AlignVCenter)
+        self.commandFormLayout.setFormAlignment(QtCore.Qt.AlignHCenter|QtCore.Qt.AlignTop)
+        self.commandFormLayout.setObjectName("commandFormLayout")
+        self.fileNameLabel = QtWidgets.QLabel(self.centralwidget)
+        self.fileNameLabel.setObjectName("fileNameLabel")
+        self.commandFormLayout.setWidget(0, QtWidgets.QFormLayout.LabelRole, self.fileNameLabel)
+        self.fileNameLineEdit = QtWidgets.QLineEdit(self.centralwidget)
+        self.fileNameLineEdit.setObjectName("fileNameLineEdit")
+        self.commandFormLayout.setWidget(0, QtWidgets.QFormLayout.FieldRole, self.fileNameLineEdit)
+        self.brokerLineEdit = QtWidgets.QLineEdit(self.centralwidget)
+        self.brokerLineEdit.setObjectName("brokerLineEdit")
+        self.commandFormLayout.setWidget(1, QtWidgets.QFormLayout.FieldRole, self.brokerLineEdit)
+        self.brokerLabel = QtWidgets.QLabel(self.centralwidget)
+        self.brokerLabel.setObjectName("brokerLabel")
+        self.commandFormLayout.setWidget(1, QtWidgets.QFormLayout.LabelRole, self.brokerLabel)
+        self.startTimeLayout = QtWidgets.QHBoxLayout()
+        self.startTimeLayout.setObjectName("startTimeLayout")
+        self.useStartTimeCheckBox = QtWidgets.QCheckBox(self.centralwidget)
+        self.useStartTimeCheckBox.setText("")
+        self.useStartTimeCheckBox.setObjectName("useStartTimeCheckBox")
```

Comment:
I'm not sure these setObjectName calls are strictly necessary - they are only used when trying to find a global QObject. If they are not necessary, may as well remove them.